<launch>
<node pkg="new_explore" type="explore_action_server" respawn="false" name="explore_action_server" output="screen">
  <param name="robot_base_frame" value="base_footprint"/>
  <param name="costmap_topic" value="map"/>
  <param name="costmap_updates_topic" value="map_updates"/>
  <param name="visualize" value="true"/>
  <param name="planner_frequency" value="10.0"/>
  <param name="progress_timeout" value="25.0"/>
  <param name="potential_scale" value="200.0"/>
  <param name="orientation_scale" value="3.0"/>
  <param name="gain_scale" value="0.25"/>
  <param name="transform_tolerance" value="0.3"/>
  <param name="min_frontier_size" value="2.0"/>
</node>
  <!-- <param output="screen"/> -->
</launch>